### ISSUE TITLE
Pain worse with pronation of the wrist

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9916]
**Preferred term label:**
Pain worse with pronation of the wrist

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Radial tunnel syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000790
- label: Exacerbated by pronation
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: 
- def: Applied to a sign or symptom that is aggravated by (or made more painful by) pronation. Pronation of the hand involves the rotation of the forearm and wrist so that the palm of the hand faces downwards or towards the back (posteriorly).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9916
- synonyms:
